### PR TITLE
MQE-2097: CHANGELOG.MD and Composer version bump

### PR DESCRIPTION
**Pull request by soumyau**
- Updated 3.0.0 RC2 changelog
- Added missing documentation

### Fixed Issues (if relevant)

1. magento/magento2-functional-testing-framework#<issue_number>: Issue title
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/verification tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)
 - [ ] Changes to Framework doesn't have backward incompatible changes for tests or have related Pull Request with fixes to tests

### CODE REVIEW

File: docs/commands/mftf.md [hunk 1]
```diff
@@ -468,14 +468,20 @@ vendor/bin/mftf static-checks testDependencies
 vendor/bin/mftf static-checks actionGroupArguments
 ```
 ```bash
+vendor/bin/mftf static-checks deprecatedEntityUsage
```
Comment (jilu1) [inline docs/commands/mftf.md:471]: Let's also add the --path option for this static check

Comment (soumyau) [reply docs/commands/mftf.md:477]: done

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,39 @@
...
+        *  Added new actions: `assertStringContainsString`, `assertStringNotContainsString`, `assertStringContainsStringIgnoringCase`, `assertStringNotContainsStringIgnoringCase` for string haystacks.
+        *  Removed actions: `assertInternalType`, `assertNotInternalType`, `assertArraySubset`.
+        *  Removed delta option from `assertEquals` and `assertNotEquals`.
+    *  Removed action `pauseExecution` and added `pause`. [See actions page for details](./docs/test/actions.md)
+    *  Removed action `formatMoney` and added `formatCurrency`. [See actions page for details](./docs/test/actions.md)    
+    *  Added new static check that checks and reports references to deprecated test entities.
+*  Bumped dependencies to support PHP/PHPUnit upgrade.
+    
+*  Traceability 
+    *  Introduced new .env configuration `VERBOSE_ARTIFACTS` to toggle saving attachments in Allure. [See configuration page for details](./docs/configuration.md)
```
Comment (KevinBKozan) [inline CHANGELOG.md:23]: Backticks \` on `.env`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,39 @@
...
+    
+*  Traceability 
+    *  Introduced new .env configuration `VERBOSE_ARTIFACTS` to toggle saving attachments in Allure. [See configuration page for details](./docs/configuration.md)  
+    
+### Fixes
+
+*  Fixed issue of resolving arguments of type `entity` in action groups within a custom helper.
+*  Fixed reporting issue in output file for `testDependencies` static check.
+*  Fixed a bug in `actionGroupArguments` static check when action group filename is missing `ActionGroup`.
+*  Fixed issue of running suites under root _suite directory in Standalone MFTF.
```
Comment (KevinBKozan) [inline CHANGELOG.md:30]: I would backtick \` `_suite`, right now the undwerscore is causing the rest of the line to be italicized.

Comment (soumyau) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (soumyau) [reply CHANGELOG.md]: fixed

File: docs/commands/mftf.md [hunk 4]
```diff
@@ -464,18 +464,31 @@ To run specific static check scripts
 ```bash
 vendor/bin/mftf static-checks testDependencies
 ```
+
 ```bash
 vendor/bin/mftf static-checks actionGroupArguments
 ```
+
+```bash
+vendor/bin/mftf static-checks actionGroupArguments -p <path_to_mftf_test_module>
```
Comment (jilu1) [inline docs/commands/mftf.md:473]: ? wrong command.

Also please indicating this is option is optional

File: docs/commands/mftf.md (hunk 1 above)
Comment (soumyau) [reply docs/commands/mftf.md:473]: rectified. Since these are examples, added 2 usages, one with and one without `-p`

**Pull request merged**